cognite/experimental/data_classes/transformations.py

```diff
@@ -0,0 +1,285 @@
+from cognite.client.data_classes._base import *
+
+
+class TransformationDestination:
+    """TransformationDestination has static properties and methods to define the target resource type of a transformation
+    """
+
+    def __init__(self, type: str = None):
+        self.type = type
+
+    @property
+    @staticmethod
+    def Assets():
+        """`To be used when the transformation is meant to produce assets.`_
+        """
+        return TransformationDestination("assets")
+
+    @property
+    @staticmethod
+    def Timeseries():
+        """`To be used when the transformation is meant to produce time series.`_
+        """
+        return TransformationDestination("timeseries")
+
+    @property
+    @staticmethod
+    def AssetHierarchy():
+        """`To be used when the transformation is meant to produce asset hierarchies.`_
+        """
+        return TransformationDestination("asset_hierarchy")
+
+    @property
+    @staticmethod
+    def Events():
+        """`To be used when the transformation is meant to produce events.`_
+        """
+        return TransformationDestination("events")
+
+    @property
+    @staticmethod
+    def Datapoints():
+        """`To be used when the transformation is meant to produce numeric data points.`_
+        """
+        return TransformationDestination("datapoints")
+
+    @property
+    @staticmethod
+    def StringDatapoints():
+        """`To be used when the transformation is meant to produce string data points.`_
+        """
+        return TransformationDestination("string_datapoints")
```

Comment:
sorry this is my bad, i documented it wrong initially. underscore should be gone.
```suggestion
        return TransformationDestination("stringdatapoints")
```